### PR TITLE
Add PipelineResourcesDropdown container

### PR DESCRIPTION
**Pull request by ncskier**
# Changes
Added a PipelineResourcesDropdown container along with its tests and a stories file.

This dropdown will be required for the manual PipelineRun paneling under issue https://github.com/tektoncd/dashboard/issues/108.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: src/containers/PipelineResourcesDropdown/PipelineResourcesDropdown.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  loading: false,
+  label: 'Select Pipeline Resource',
+  titleText: 'Pipeline Resource'
+};
+
+function mapStateToProps(state, ownProps) {
+  const namespace = ownProps.namespace || getSelectedNamespace(state);
+  return {
+    items: getPipelineResources(state, { namespace }).map(
+      sa => sa.metadata.name
```
Comment (AlanGreene) [inline src/containers/PipelineResourcesDropdown/PipelineResourcesDropdown.js]: I'm guessing `sa` was for service account, should update the variable name to `pipelineResource` or similar

File: src/containers/PipelineResourcesDropdown/PipelineResourcesDropdown.stories.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      name: 'pipeline-resource-3',
+      namespace: 'default',
+      uid: 'id-pipeline-resource-3'
+    }
+  }
+};
+
+const namespacesByName = {
+  default: '',
+  'kube-public': '',
```
Comment (AlanGreene) [inline src/containers/PipelineResourcesDropdown/PipelineResourcesDropdown.stories.js]: no need for the other namespaces, just `default`

File: src/containers/PipelineResourcesDropdown/PipelineResourcesDropdown.js (hunk 1 above)
Comment (ncskier) [reply src/containers/PipelineResourcesDropdown/PipelineResourcesDropdown.js]: Yep, good catch! 😅

**Comment by ncskier** [comment]
Thanks for the review @AlanGreene. I just addressed your change requests; let me know what you think 🙂

**Pull request merged**